packages/foam-vscode/src/features/link-completion.ts

```diff
@@ -197,7 +197,20 @@ export class CompletionProvider
       item.range = replacementRange;
       item.command = COMPLETION_CURSOR_MOVE;
       item.commitCharacters = linkCommitCharacters;
-      return item;
+      const aliasItems = resource.aliases.map(a => {
```

Comment:
I wonder if we should just have
```
const resources = ...
const aliases = ...
...
return new vscode.CompletionList([...resources, ...aliases, ...placeholders]);
```

This way we are sure all resources are shown before the resources